### PR TITLE
Release v1.4.8

### PR DESCRIPTION
**Pull request by matzebond**
## Important changes
- include a complete Dutch translation

## Minor changes
- migrate dart to nulsafty & update dependencies
- update theme code

----

test `test` 

``` dart
int? a = b ? 0
```

#

### CODE REVIEW

**Comment by matzebond** [comment]
a further test

``` dart
int? a = b ? 0;
```

File: lib/helpers/lang_helper.dart [hunk 1]
```diff
@@ -0,0 +1,192 @@
...
+  "xh": "isiXhosa",
+  "yi": "ייִדיש",
+  "yo": "èdè Yorùbá",
+  "za": "Vahcuengh / 話僮",
+  "zh": "中文 Zhōngwén",
+  "zu": "isiZulu"
+};
+
+// languages not in this list will be marked as BETA
+final List<String> completeLanguages = [
```
Comment (dfuchss) [inline lib/helpers/lang_helper.dart:174]: @matzebond you may have to add the new language here as well ..

Comment (dfuchss) [inline lib/helpers/lang_helper.dart:174]: All Languages synced here?

Comment (matzebond) [reply lib/helpers/lang_helper.dart:174]: In weblate currently Norwegian Bokmål, Greek and Esperanto are only partially translated.

**Comment by matzebond** [comment]
Closing this as a merge into master is not necessary for a f-droid release.

**Pull request closed without merging**